camera-orderfile/basic/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C)  2017 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cmake_minimum_required(VERSION 3.4.1)
```

Comment:
Wow, I wonder why any sample every had this. Android has never supported a CMake this old. Might as well make this 3.22.1.